src/edit/options.js

```diff
@@ -132,6 +132,7 @@ export function defineOptions(CodeMirror) {
     }
     cm.display.input.readOnlyChanged(val)
   })
+  option("srLabel", 'CodeMirror text input')
```

Comment:
I think I'd prefer to call this option either just `label` or `screenReaderLabel`. The abbreviation seems unnecessary.